origami.json

```diff
@@ -0,0 +1,37 @@
+{
+	"description": "Maintain consistent spaces across components and projects in accordance to our baseline grid and design guidelines.",
+	"keywords": ["whitespace", "spacing"],
+	"origamiType": "module",
+	"origamiCategory": "components",
+	"origamiVersion": 1,
+	"support": "https://github.com/Financial-Times/o-spacing/issues",
+	"supportContact": {
+		"email": "origami.support@ft.com",
+		"slack": "financialtimes/ft-origami"
+	},
+	"supportStatus": "experimental",
+	"browserFeatures": {},
+	"ci": {
+		"circle": "https://circleci.com/api/v1/project/Financial-Times/o-spacing"
+	},
+	"demosDefaults": {
+		"sass": "demos/src/demo.scss",
+		"documentClasses": "o-typography-wrapper",
+		"dependencies": ["o-normalise", "o-typography", "o-colors"]
```

Comment:
Small reminder to pin demo dependencies ✌️

Reply:
Oops, good catch 👍